docroot/modules/humsci/hs_paragraph_types/modules/hs_hero_image/config/install/responsive_image.styles.hero_image.yml

```diff
@@ -5,42 +5,38 @@ dependencies:
     - image.style.hero_large
     - image.style.hero_medium
     - image.style.hero_small
-  theme:
-    - stanford_basic
-_core:
-  default_config_hash: _ldYlaDS02eaxI53rCam6aU1VEhTWvjxNE4ZsLrd9Xo
 id: hero_image
 label: 'Hero image'
 image_style_mappings:
   -
-    breakpoint_id: stanford_basic.lg
+    breakpoint_id: stanford_media.lg
```

Comment:
New dependency on stanford_media that isn't captured?

Reply:
https://github.com/SU-HSDO/suhumsci/blob/develop/docroot/modules/humsci/hs_paragraph_types/modules/hs_hero_image/hs_hero_image.info.yml#L11

Reply:
Yeah, but this is a different config and has it's own dependencies list. It is missing.

Reply:
💥 got it 👍 https://github.com/SU-HSDO/suhumsci/pull/14/commits/9e899f9ff15744219b4d2be75f75aebccf07355e